README.md

```diff
@@ -45,9 +45,9 @@ You should split your API into categories. Each of those categories should have
 Example `Api/Stats`. The response of any call should be an object in `Resource/Api/X`. Example 
 `Resource/Api/Stats/TotalResponse`.
 
-### Deserialzier
+### ResponseHydrator
```

Comment:
Yes, It also looks better with `ArrayHydrator` if the interface is just called `Hydrator`. 

Because you are also talking about the interface, right? Not just the header in the readme?